src/qt/bitcoingui.cpp

```diff
@@ -701,6 +707,20 @@ void BitcoinGUI::openClicked()
     }
 }
 
+void BitcoinGUI::openWalletClicked()
+{
+    QString fileName = QFileDialog::getExistingDirectory(this, tr("Open Wallet"),
+                                                QString(),
+                                                QFileDialog::ShowDirsOnly
+                                                | QFileDialog::DontResolveSymlinks);
+    if (fileName.isEmpty()) return;
+    const auto& result = m_node.loadWallet(fileName.toStdString());
```

Comment:
The use of `auto` here makes it difficult to understand the `result.second` below.

Reply:
No an issue now, return value is not necessary.